src/mina-signer/src/sign-zkapp-command.unit-test.ts

```diff
@@ -0,0 +1,297 @@
+import { expect } from 'expect';
```

Comment:
this is a file that was renamed. actual changes: https://github.com/o1-labs/snarkyjs/pull/717/commits/2b946d4ec035affc0503614d68c5ef91b2efc58c and https://github.com/o1-labs/snarkyjs/pull/717/commits/1fd93c63112b14787cc4af9fbab7862d112c6c84